github_pr.py

```diff
@@ -42,6 +43,8 @@ def _print_pr(pr, **args):
     elif 'comments' in args and args['comments']:
         for comment in pr.get_comments():
             print "#%d : Comment - %s" % (pr.number, comment.body)
+    elif 'filters' in args and args['filters']:
+        print "%d" % (pr['pr'].number)
```

Comment:
ok, thanks @ferrants. @babineaum I think that list doesn't conditionally depend on anything we're filtering for (all those attributes will always exist on the PR object we have) - sounds like a good default set of columns to print?